VMBackup/main/fsfreezer.py

```diff
@@ -150,7 +171,7 @@ def freeze_safe(self,timeout):
             self.logger.log(errMsg,True,'Warning')
         try:
             freeze_result = FreezeResult()
-            freezebin=os.path.join(os.getcwd(),os.path.dirname(__file__),"safefreeze/bin/safefreeze")
+            freezebin=os.path.join(os.getcwd(),os.path.dirname(__file__),self.safeFreezeFolderPath)
```

Comment:
Can we also test the error we get when we run wrong architecture executable. And based on that can we fallback to the other one ?

Reply:
actually, when we create the child process, we don't check whether that succeed....we were waiting for it to complete but it is getting crashed. it is tough to handle in such a way